setup.py

```diff
@@ -30,7 +30,7 @@
 extras_require = {
     'datatype': [
         'beautifulsoup4>=4.6',
-        'h5py>=2.7',
+        'h5py>=2.9',
```

Comment:
Do we need new features from h5py for this to work?

Reply:
The answer is yes.

Reply:
yes!